themes/default/templates/nav.html

```diff
@@ -65,6 +65,7 @@ <h3 data-bind="text: name, typeset: name"></h3>
         <ul class="nav">
             <li><button type="button" class="btn navbar-btn btn-primary" data-bind="click: Numbas.controls.previousQuestion, attr: {disabled: !canReverse()}"><span class="sr-only" data-localise="control.previous"></span>&larr;</button></li>
             <li><button type="button" class="btn navbar-btn btn-primary" data-bind="click: Numbas.controls.nextQuestion, attr: {disabled: !canAdvance()}"><span class="sr-only" data-localise="control.next"></span>&rarr;</span></button></li>
+            <li><button class="btn btn-primary navbar-btn center-block" id="endBtn" data-localise="control.end exam" data-bind="click: Numbas.controls.endExam, visible: mode()=='normal'"></button></li>
```

Comment:
Remove the `id` attribute - you shouldn't have two elements in the page with the same ID, and I'd prefer that the one on the sidebar gets the ID.